### ISSUE TITLE
Qusetions about the test resutls on SALICON test set.

### ISSUE THREAD

**Issue by pengqianli** [samyak0210/saliency#4]
In this paper, the author said "The model trained on SALICON was finetuned using MIT1003 and CAT2000." I am wondering that if the model is fine-tuned on MIT1003 and CAT2000 when generating the resutls on SALICON 2017 test set? 
In  short: 
    on SALICON: model trained on SALICON -> test on SALICON.
    on MIT300: model trianed on SALICON+MIT1003 -> test on SALICON.
or like that
    on SALICON: model trained on SALICON+MIT1003+CAT2000 -> test on SALICON.
    ...

**Comment by samyak0210** [comment]
Hey @pengqianli, for SALICON test the model is trained on SALICON and tested on SALICON test. For MIT1003, model trained on SALICON is finetuned on MIT1003(k-fold cross validation) and tested on MIT300 test set. Similarly for CAT2000.

So,
on SALICON: model trained on SALICON -> test on SALICON
on MIT300: model trained on SALICON + MIT1003(finetuned using k-fold cross validation) -> test on MIT300

Does this solve your issue?

**Comment by pengqianli** [comment]
yes, thanks!
